### PR TITLE
Reorganize api and mocks

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This organizes our api and mocks into the services folder, partly for clarity and partly to prepare for the new future vaos api iteration. Definitely open for feedback on if there are better ways of organizing this.

Changes:
- Moved api/index.js to services/var/index.js. VAR is the prefix for most of these requests we make in VAMF
- Moved mock files in api/ to services/mocks/var
- Moved mock files for fhir apis to services/mocks/fhir
- Created a single handlers file for both sets of apis in services/mocks

## Testing done
Unit testing

## Screenshots
Screen Shot 2020-09-03 at 11 58 23 AM

## Acceptance criteria
- [ ] Mock organization acceptable to team

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by vbahinwillit — COMMENTED** [review]
LGTM

File: src/applications/vaos/tests/e2e/vaos-cypress-helpers.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-import slots from '../../api/slots.json';
+import confirmedVA from '../..//services/mocks/var/confirmed_va.json';
+import confirmedCC from '../..//services/mocks/var/confirmed_cc.json';
+import requests from '../..//services/mocks/var/requests.json';
+import cancelReasons from '../..//services/mocks/var/cancel_reasons.json';
+import supportedSites from '../..//services/mocks/var/sites-supporting-var.json';
+import facilities from '../..//services/mocks/var/facilities.json';
+import facilities983 from '../..//services/mocks/var/facilities_983.json';
+import clinicList983 from '../..//services/mocks/var/clinicList983.json';
+import slots from '../..//services/mocks/var/slots.json';
```
Comment (narin) [inline src/applications/vaos/tests/e2e/vaos-cypress-helpers.js:12]: Extra slashes here

File: src/applications/vaos/tests/e2e/vaos-helpers.js [hunk 2]
```diff
@@ -4,16 +4,16 @@ const mock = require('../../../../platform/testing/e2e/mock-helpers');
...
+const confirmedVA = require('../..//services/mocks/var/confirmed_va.json');
+const confirmedCC = require('../..//services/mocks/var/confirmed_cc.json');
+const requests = require('../..//services/mocks/var/requests.json');
+const cancelReasons = require('../..//services/mocks/var/cancel_reasons.json');
+const supportedSites = require('../..//services/mocks/var/sites-supporting-var.json');
+const facilities = require('../..//services/mocks/var/facilities.json');
+const facilities983 = require('../..//services/mocks/var/facilities_983.json');
+const clinicList983 = require('../..//services/mocks/var/clinicList983.json');
+const slots = require('../..//services/mocks/var/slots.json');
+const pact = require('../..//services/mocks/var/pact.json');
```
Comment (narin) [inline src/applications/vaos/tests/e2e/vaos-helpers.js:16]: Same thing here with the slashes and I think most of the other specs

**Review by narin — APPROVED** [review]
LGTM.  Just some files with extra slashes in the imports.

**Pull request merged**